support/tech-resources/smtp-rest-api-performance.md

```diff
@@ -9,13 +9,16 @@ SparkPost wants to make sure all of our customers have the best sending experien
 
 We recommend that you send via the Transmission REST API, if possible. Overall the REST API is more efficient than SMTP and requires less resources on the client side.
 
-* Keep the size of the transmission to approximately 50MB.  This refers to the total size of the JSON used for transmission, including any substitution data.
+* On timeout or any 5XX error the best practice is to retry.
+* Set API timeout to 300 seconds.
+* There is a 300MB limit on payload size for each transmission, but we recommend keeping a payload size no greater than 50MB.  This refers to the total size of the JSON used for transmission, including any substitution data.
```

Comment:
Tempted to also explicitly state this includes recipients (thinking multi-recip flow).  What do you think?

Reply:
Good idea, will do